x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsUuidValidationIntegTests.java

```diff
@@ -33,6 +34,7 @@
 import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked;
 import static org.hamcrest.Matchers.containsString;
 
+@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST)
```

Comment:
There is a single test in this class, why changing the scope to TEST?

Reply:
It's because when you run the test with `tests.iters=x`, it will reuse the same node, and that breaks the test (RestoreBlockingActionFilter.executed and unblocked are not reset)